pegasus/sites.v3/code.org/views/music/music_lab_featured_artists.haml

```diff
@@ -0,0 +1,8 @@
+- artist_index = 5
+
+- artist_photo = ["/images/music-lab/chainsmokers.png", "/images/music-lab/rosa-linn.png", "/images/music-lab/tinashe.png", "/images/music-lab/dove-cameron.png", "/images/music-lab/aloe-blacc.png"]
+
+%ul.featured-artists.clear-styles.flex-container.justify-space-between.wrap
+  - artist_index.times do |index|
+    %li
+      %img{src: artist_photo[index], alt: ""}
```

Comment:
I removed the alt text here since there will be a list of artists next to this and it felt redundant — happy to add it back if y'all think we need it.